src/mysql/populate.sql

```diff
@@ -13,7 +13,7 @@ VALUES
   ('ITI'),
```

Comment:
nit - if this file is populating dummy data, that should be clear by the filename. Either it should be included in a testdata directory of the filename should reference that it is populating dummy data.

Reply:
Same, I'll make SQL changes in another PR.

Reply:
If you're going to wait on those SQL changes can you add TODOs?

Reply:
Sure!